app/assets/stylesheets/components/profile-form.scss

```diff
@@ -0,0 +1,38 @@
+.profile-form {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
```

Comment:
Properties should be ordered display, flex-direction, max-width, position, text-align, width